[New] Changelog for API Spec 4.6.0

leslitagordita opened the pull request:
* Add changelog file for API spec release 4.6.0
* Add changelog entry for `linode_reboot` secondary_entity

leslitagordita committed:
Add changelog file for API spec release 4.6.0

Add changelog entry for `linode_reboot` secondary_entity

leslitagordita committed:
Update copy for secondary_entity changelog entry.

Commit:
added managed-stats endpoint log entry

andystevensname committed:
Added fixed entry for filtering and sorting examples

Commit:
updated managed-stats endpoint entry

andystevensname committed:
Added fixed entry for preventing IPv6 /116 pool migrations

andystevensname committed:
Improved IPv6 /116 pools entry

andystevensname committed:
Added Object Storage changes to Breaking header

yoshizzle commented on src/content/changelog/2019-10-07-api.md:
```diff
@@ -0,0 +1,27 @@
+
+* Fixed a formatting error that prevented the [filtering and sorting](https://developers.linode.com/api/v4/#filtering-and-sorting) examples within the API documentation from correctly working as presented.
+
+* Fixed an issue that allowed Linodes with IPv6 /116 pools to attempt to migrate to Next Generation Network (NGN) enabled data centers like Toronto and Mumbai. IPv6 /116 pools are not supported by NGN data centers, so these migrations would never actually progress. A Linode with IPv6 /116 pools will now be prevented from initiating a migration to an NGN data center, and the user will be prompted  ...
```
datacenters is one word, not two. There are two places in here where it's separated.

andystevensname committed:
Fixed spelling of datacenters

andystevensname committed:
Data center to datacenter

andystevensname committed:
Datacenter back to data centers

After consulting our internal style guide I'm reverting these back to two words.
